### PR TITLE
Bug fix/multiple index in setup each

### PR DESCRIPTION
**Pull request by rueyjye**
Create PR again
https://github.com/lifull-dev/bucky-core/pull/65

### CODE REVIEW

File: lib/bucky/core/test_core/test_class_generator.rb [hunk 1]
```diff
@@ -16,9 +16,9 @@ module TestClassGeneratorHelper
         def add_test_procedure(procedures)
           procedures.each.with_index(1) do |procedure, index|
             procedure[:proc] ||= ''.dup
-            puts procedure[:proc].insert(0, "  #{index}:") if procedure.key?(:proc)
+            puts "  #{index}:#{procedure[:proc]}"
             method = procedure[:exec].key?(:operate) ? :operate : :verify
-            send(method, procedure)
+            send(method, exec: procedure[:exec], index: index)
```
[the remark this answers is not in the document]
Comment (rueyjye) [reply lib/bucky/core/test_core/test_class_generator.rb]: I'll change to step_number

File: lib/bucky/core/test_core/test_class_generator.rb [hunk 2]
```diff
@@ -18,7 +18,7 @@ def add_test_procedure(procedures)
             procedure[:proc] ||= ''.dup
             puts "  #{index}:#{procedure[:proc]}"
```
Comment (naotospace) [inline lib/bucky/core/test_core/test_class_generator.rb]:
```suggestion
            puts "  #{step_number}:#{procedure[:proc]}"
```

Comment (naotospace) [reply lib/bucky/core/test_core/test_class_generator.rb]: ` procedures.each.with_index(1) do |procedure, index|`
-> ` procedures.each.with_index(1) do |procedure, step_number|`

File: lib/bucky/core/test_core/test_class_generator.rb [hunk 3]
```diff
@@ -18,7 +18,7 @@ def add_test_procedure(procedures)
...
             method = procedure[:exec].key?(:operate) ? :operate : :verify
-            send(method, exec: procedure[:exec], index: index)
+            send(method, exec: procedure[:exec], step_number: index)
```
Comment (naotospace) [inline lib/bucky/core/test_core/test_class_generator.rb]:
```suggestion
            send(method, exec: procedure[:exec], step_number: step_number)
```

File: lib/bucky/core/test_core/test_class_generator.rb (hunk 2 above)
Comment (rueyjye) [reply lib/bucky/core/test_core/test_class_generator.rb]: fixed!

File: lib/bucky/core/test_core/test_class_generator.rb (hunk 3 above)
Comment (rueyjye) [reply lib/bucky/core/test_core/test_class_generator.rb]: fixed!

**Pull request merged**